### PR TITLE
ETC - Corrupted Standard, Transport Magic Circle

### PR DESCRIPTION
**Pull request by vtange**

### CODE REVIEW

File: ETC.tsv [hunk 1]
```diff
@@ -10375,7 +10375,7 @@ ETC_20150323_010376	Detect strange aura
 ETC_20150323_010377	{memo X}The place to bury the bottle
 ETC_20150323_010378	Metiken appearance trigger
 ETC_20150323_010379	{memo X}The place to install the magic field
-ETC_20150323_010380	Polluted pile of soil
+ETC_20150323_010380	Corrupted Pile of Soil
```
Comment (ttgmichael) [inline ETC.tsv]: You sure it's a corrupted pile of soil? there's somes quest lines I saw about piles of soil

Comment (vtange) [inline ETC.tsv]: iirc there are "corrupted/evil soil" in places like Gele Plateau (where you drag this doll thing around)

Comment (terrapin47) [inline ETC.tsv]: That one is currently referred to as "demon corrupted land"

Comment (vtange) [inline ETC.tsv]: aight. better safe than sorry

**Comment by Sunaries** [comment]
:+1:

**Comment by imcgames** [comment]
Thank you :smile:

**Pull request merged**